labs-ios-starter/Search/SearchViewController.swift

```diff
@@ -10,26 +10,52 @@ import UIKit
 import MapKit
 
 class SearchViewController: UIViewController {
+    
     // MARK: - Properties
+    
     var searchResponse = Map()
     var network = NetworkClient()
     var city = ""
     var state = ""
     
     // MARK: Outlets
+    
     @IBOutlet weak var backgroundGradient: UIView!
     @IBOutlet weak var searchBar: UISearchBar!
     
     override func viewDidLoad() {
         super.viewDidLoad()
         setGradientBackgroundColor()
-        // Do any additional setup after loading the view.
     }
     
-    // MARK: - Background gradient
+    // MARK: - Navigation
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "toMap" {
+            let vc = segue.destination as! MapScreenViewController
+            vc.searchItem = searchResponse
+            
+            network.getWalkability(city: city, state: state) { (walkability, error) in
+                if error != nil {
+                    DispatchQueue.main.async {
+                        vc.performSegue(withIdentifier: "unwindToSearch", sender: self)
+                    }
+                    return
+                }
```

Comment:
Might be good to add some logging about the error that took place. You don't have a logging system set up, but you could just do NSLog so at lease we are doing something with the possible error we receive from getting the walkability.